src/sdk/pynni/nni/hyperopt_tuner/hyperopt_tuner.py

```diff
@@ -61,6 +62,12 @@ def json2space(in_x, name=ROOT):
             if _type == 'choice':
                 out_y = eval('hp.hp.'+_type)(name, _value)
             else:
+                if _type in ['loguniform', 'qloguniform']:
+                    if len(_value) == 3:
+                        _value = list(map(lambda x: math.log(x, _value[-1]), _value[:2]))
+                    else:
+                        _value = np.log10(_value[:2])
+                    _type = 'uniform'
```

Comment:
I don't think you could use uniform to instead qloguniform, because q is a step number, it will constraint the value be sampled.